plop-templates/react.tsx.hbs

```diff
@@ -8,8 +8,7 @@ import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from 'rea
 
 export interface {{pascalCase name}}Props extends PropsWithChildren<HTMLAttributes<HTMLElement>> {}
 
-export const {{pascalCase name}} = forwardRef(
-  ({ children, className, ...restProps }: {{pascalCase name}}Props, ref: ForwardedRef<HTMLElement>) => (
+export const {{pascalCase name}} = forwardRef(({ children, className, ...restProps }: {{pascalCase name}}Props, ref: ForwardedRef<HTMLElement>) => (
```

Comment:
This is only correct for components with names of 5 characters or shorter. I think most component names will be longer than that, so I wouldn't change this.